spec/money_spec.rb

```diff
@@ -38,6 +38,10 @@
     expect(money.to_money).to eq(money)
   end
 
+  it "#to_money uses the provided currency when it doesn't already have one" do
```

Comment:
Maybe add test case for `to_money` on `Money` with currency to verify that it doesn't override?

Reply:
good call, done